storj/model.py

```diff
@@ -250,6 +253,77 @@ def address(self):
         return self.keypair.address(use_uncompressed=False)
 
 
+class IdecdsaCipher(Object):
+    """
+    Tools for en-/decrypt private key to/from id_ecdsa
+    """
+
+    def __init__(self):
```

Comment:
you don't need to declare the `__init__()` function if you're just going to `pass` :)